nullaway/src/test/java/com/uber/nullaway/jspecify/ArrayTests.java

```diff
@@ -608,6 +608,25 @@ public void mismatchedIndexUse() {
         .doTest();
   }
 
+  @Test
+  public void typeUseAndDeclarationLegacyAnnotationOnArray() {
```

Comment:
This is not testing legacy mode.  In fact, is there a way for us to fail if the config says that both JSpecify mode and legacy mode are enabled?  That shouldn't be allowed.

Reply:
Our JSpecify tests run fine when both are enabled. Do we still want to restrict this?

Reply:
@armughan11 I think yes, since it's just invalid, and probably the user made a mistake.